deepchecks/__init__.py

```diff
@@ -96,17 +96,12 @@
     __version__ = ''
 
 
-# Check for latest version
+# Send an import event
 try:
-    disable = os.environ.get('DEEPCHECKS_DISABLE_LATEST', 'false').lower() == 'true'
+    disable = os.environ.get('DISABLE_DEEPCHECKS_ANONYMOUS_TRACKING', False)
```

Comment:
there is already if disabled inside the function
```suggestion
    send_anonymous_import_event()
```